lib/utils/ProductQueryConfigurations.dart

```diff
@@ -2,15 +2,24 @@ import 'package:openfoodfacts/utils/AbstractQueryConfiguration.dart';
 import 'package:openfoodfacts/utils/CountryHelper.dart';
 import 'package:openfoodfacts/utils/LanguageHelper.dart';
 import 'package:openfoodfacts/utils/ProductFields.dart';
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:openfoodfacts/utils/UriHelper.dart';
+
+enum ProductQueryVersion {
+  V0,
+  V2,
```

Comment:
We should somewhere document what the difference between those two are. 

I don't even know what separates them

Reply:
Good point.
I don't know either the difference - except that ingredients look better on V2 (tree instead of flat structure), and that knowledge panels used V2.

Reply:
Actually v2048 will work 🙃
There is some documentation in the official reference, but it might not be complete. I remember that the products route is a major one, and possibly ingredients nesting